mlrun/wasbfs/fs.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2022 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from urllib.parse import urlparse
+
+from adlfs import AzureBlobFileSystem
+from fsspec import AbstractFileSystem
+
+
+class WasbFS(AbstractFileSystem):
+
+    protocol = "wasb"
+
+    def __init__(
+        self,
+        account_name: str = None,
+        account_key: str = None,
+        connection_string: str = None,
+        credential: str = None,
+        sas_token: str = None,
+        request_session=None,
+        socket_timeout: int = None,
+        blocksize: int = None,
+        client_id: str = None,
+        client_secret: str = None,
+        tenant_id: str = None,
+        anon: bool = True,
+        location_mode: str = None,
+        loop=None,
+        asynchronous: bool = False,
+        default_fill_cache: bool = True,
+        default_cache_type: str = None,
+        **kwargs,
+    ):
+
+        self.azureBlobFS = AzureBlobFileSystem(
+            account_name,
```

Comment:
we talked about it offline. I think you should add a comment why you decided to implement it as an attribute of the class and not to inherit from `AzureBlobFileSystem` as you are just wrapping some of its methods.